stackslib/fuzz/fuzz_targets/bitcoin_blocks_parser_parse_tx_raw.rs

```diff
@@ -0,0 +1,52 @@
+#![no_main]
```

Comment:
Nit: please include the copyright code block. Thanks!

Reply:
Thanks! Done in f465f9e16cac952815336d483e9d9358ee43b96d.